src/builtins/amp-layout/build-dom.js

```diff
@@ -0,0 +1,24 @@
+import {Layout, applyFillContent} from '#core/dom/layout';
+import {realChildNodes} from '#core/dom/query';
+
+import {getEffectiveLayout} from '../../static-layout';
```

Comment:
In general, would it make sense to use our existing import restriction eslint rules to disallow `**/build-dom.js` from importing anything outside of `#core`?

In this case `getEffectiveLayout` can be cored eventually and this can be excluded for now, but more generally speaking it might help avoid anyone introducing bad imports

Reply:
Yes. Yes it would.